powerline_shell/themes/default.py

```diff
@@ -69,6 +69,9 @@ class DefaultColor(object):
     AWS_PROFILE_FG = 39
     AWS_PROFILE_BG = 238
 
+    TIME_FG = 15
```

Comment:
Thanks for looking into this. You mean to say that here it should default to 250, 238 respectively?
Similar to 
```
 HOSTNAME_FG = 250
 HOSTNAME_BG = 238
```

Reply:
Yes exactly. I am always wary of making changes that will affect existing users and force them to take some action if they don't like the new change.